embeddings/pipeline/lightning_classification.py

```diff
@@ -57,7 +57,7 @@ def __init__(
             dataloader_kwargs=config.dataloader_kwargs,
             **config.datamodule_kwargs
         )
-        task = TextClassificationTask(
+        task = TextClassificationTaskClassification(
```

Comment:
why renamed?

Reply:
Not intended change during refactor. Reverted